guides/source/active_record_basics.md

```diff
@@ -185,20 +230,20 @@ can use the `ActiveRecord::Base.table_name=` method to customize the table name
 that should be used:
 
 ```ruby
-class Product < ApplicationRecord
-  self.table_name = "my_products"
+class Book < ApplicationRecord
+  self.table_name = "my_books"
 end
 ```
 
 If you do so, you will have to manually define the class name that is hosting
-the fixtures (`my_products.yml`) using the `set_fixture_class` method in your
+the fixtures (`my_books.yml`) using the `set_fixture_class` method in your
 test definition:
```

Comment:
Either in this paragraph or following the code snippet, we should link to the [Testing Rails Applications](https://guides.rubyonrails.org/testing.html) and specifically [The Low Down On Fixtures](https://guides.rubyonrails.org/testing.html#the-low-down-on-fixtures) section.

In the paragraph:

```
If you do so, you will have to manually define the class name that is hosting 
[the fixtures](https://guides.rubyonrails.org/testing.html#the-low-down-on-fixtures) (`my_books.yml`) 
using the `set_fixture_class` method in your test definition:
```

After the code block:
`NOTE: To learn more about testing and fixtures, read the guide on [Testing Rails Applications](https://guides.rubyonrails.org/testing.html)`